.travis.yml

```diff
@@ -22,15 +17,22 @@ stages:
 matrix:
   include:
     - stage: *stage_sanity_test
+      sudo: false
+      python: 3.6
       script:
       - make ci-test-unit
 
     - stage: *stage_functional
+      sudo: required
```

Comment:
Why do you need sudo in the first place?

Reply:
Oh, you already have in the stages ordering section, so nevermind.

Reply:
Can it run docker without sudo?

Reply:
It looks like Travis does this implicitly:

> Travis CI automatically routes builds to run on our Trusty sudo-enabled infrastructure when services: docker is configured. We do not currently support use of Docker on OS X.

Reply:
Yea, I can't plug kafka to a sudo: false build either.